jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBPreparedInsertionStatement.java

```diff
@@ -49,7 +49,7 @@ public boolean execute() throws SQLException {
       req.unsetMeasurements();
       req.unsetTimestamp();
       req.unsetValues();
-      return resp.getStatus().getStatusCode() == TS_StatusCode.SUCCESS_STATUS;
+      return resp.getStatus().getStatusType().getCode() == SUCCESS_CODE;
```

Comment:
```suggestion
      return resp.getStatus().getStatusType() == TSStatusType.SUCCESS_CODE;
```

Reply:
Then, you can remove "private static final int SUCCESS_CODE = 200;" 
It's better to use TSStatusType instead of defining an int in each class.